CHANGES/4057.bugfix

```diff
@@ -0,0 +1 @@
+Update multidict requirement to >= 4.5
```

Comment:
Should it be a bugfix or a feature fragment?

Reply:
Oh, it should probably be a feature for master but bugfix in backport.

Reply:
The PR will be backported to 3.6 for making 3.6.1 release.
After merging back 3.6 branch (after making the release) into the master changelog entry disappears, it will be recorded in 3.6.1 section for CHANGES.rst

Reply:
Still, it should be listed under removals in 4.0

Reply:
Why? It is a bugfix for the latest release.
There is no matter what the next planned version is; it is the bugfix anyway. 
It is neither a new feature, not removal but exactly a bugfix. 
Which I want to backport to 3.6 branch.

Reply:
But it's not a bugfix for 4.0 because it's not released yet. My point is that this shouldn't be bugfix in master in the first place.

Reply:
I cannot buy your argument, sorry.
Let's assume we don't plan 3.6.1 bugfix release, but the next aiohttp release will be 4.0.
If 4.0 fixes a problem that was introduced in 3.6 -- it is definitely the bugfix, why not?

Reply:
There is a bug; and there is a fix of the bug.
Why the fix cannot be named "bugfix"?
What did I miss?

Reply:
fair enough